lib/matplotlib/tests/test_axes.py

```diff
@@ -4540,6 +4540,29 @@ def test_text_labelsize():
     ax.tick_params(direction='out')
 
 
+@image_comparison(baseline_images=['pie_default'], extensions=['png'])
+def test_pie_default_1():
+    # The slices will be ordered and plotted counter-clockwise.
+    labels = 'Frogs', 'Hogs', 'Dogs', 'Logs'
+    sizes = [15, 30, 45, 10]
+    colors = ['yellowgreen', 'gold', 'lightskyblue', 'lightcoral']
+    explode = (0, 0.1, 0, 0)  # only "explode" the 2nd slice (i.e. 'Hogs')
+    fig1, ax1 = plt.subplots(figsize=(8, 6))
+    pie1 = ax1.pie(sizes, explode=explode, labels=labels, colors=colors,
+                autopct='%1.1f%%', shadow=True, startangle=90)
+
+
+@image_comparison(baseline_images=['pie_default'], extensions=['png'])
+def test_pie_default_2():
```

Comment:
Why the two test? Is it for ax.pie vs pyplot.pie? I'm not sure if we need both. `pyplot.pie` is just a very thin wrapper around `Axes.pie`. Maybe someone with more experience on matplotlib tests can comment on this.

If both should stay, please name them semantically, e.g. ..._axes, ..._pyplot instead of just numbering them.

Reply:
yes it's axes.pie vs pyplot.pie, I added 2 tests because sometimes it might pass the pyplot.pie test and fail the axes.pie test (this is the case if you do plt.axis("image") in the pyplot.pie) , but I am not sure if its always the case that if axes.pie passes then pyplot.pie must also pass.